services/statestorage/adapter/memory_persistence_test.go

```diff
@@ -8,16 +8,16 @@ import (
 var _ = Describe("Reading a Key", func() {
 	When("not providing a contract name", func() {
 		It("Returns an error", func() {
-			d := NewInMemoryStatePersistence(&struct{}{})
-			_, err := d.ReadState("")
+			d := NewInMemoryStatePersistence()
```

Comment:
Recommend that you kill ginkgo as soon as you can

Reply:
before it hatches